docs/onboarding/glossary.md

```diff
@@ -1,7 +1,168 @@
 ---
-id: iron-fish-glossary
-title: Glossary / FAQ
-sidebar_label: Glossary / FAQ
-description: Glossary / FAQ | Iron Fish Onboarding
-hide_table_of_contents: true
+id: glossary
+title: Glossary
+sidebar_label: Glossary
+description: Glossary | Iron Fish Onboarding
+hide_table_of_contents: false
 ---
+
+## Asset
+
+A user-created resource which can be transferred privately, minted, and burned. Contains metadata, a name, the owner public address, and nonce. 
+
+### Asset Generator
+
+The [`group_hash`](#group-hash) of the [Hashed Asset Info](#hashed-asset-info). Calculated in public so the hash can be verified at consensus.
+
+### Asset Identifier
+
+The 32-byte representation of the [Asset Generator](#asset-generator) point.
+
+### Hashed Asset Info
+
+A Pedersen Hash of the Asset Info (owner, name, metadata, nonce) . This hash is the pre-image for the [`group_hash`](#group-hash) to generate the [Asset Generator](#asset-generator) point.
+
+## Block
+
+A piece of data in the Iron Fish chain which holds a record of transactions, graffiti, difficulty, previous [Block](#block), sequence, and timestamp.
+
+### Graffiti
+
+32 byte field that may be assigned at will by the miner who mined the block.
+
+### Difficulty
+
+Measure of how difficult it is to produce an Iron Fish [Block](#block).
+
+### Sequence
+
+The sequential index number of a block in the Iron Fish chain, or the number of blocks preceding a given [Block](#block).
```

Comment:
> the number of blocks preceding a given [Block](#block).

I think this might be slightly off -- I'd think of it like the number of blocks up to and including the current block (since the genesis sequence is 1)